docs/user-guide/configuring/ignoring-code.md

```diff
@@ -0,0 +1,182 @@
+# Ignoring Code
+
+
+* [`ignorePatterns` in Config Files](#ignorepatterns-in-config-files)
+* [The `.eslintignore` File](#the-eslintignore-file)
+* [Using an Alternate File](#using-an-alternate-file)
+* [Using eslintIgnore in package.json](#using-eslintignore-in-packagejson)
+* [Ignored File Warnings](#ignored-file-warnings)
+* [Disabling Inline Comments](#disabling-inline-comments)
+
+## `ignorePatterns` in Config Files
+
+You can tell ESLint to ignore specific files and directories using `ignorePatterns` in your config files. Each value of `ignorePatterns` is a glob pattern similar to what you would use on the command line.
```

Comment:
I think this is a wrong explanation because the patterns in `ignorePatterns` are different from the glob patterns on the command line. The patterns are intended to be the same as `.gitignore`-style patterns, as same as `.eslintignore`.

Therefore, what the `The `.eslintignore` File` section described matches the `ignorePatterns` property. For example, it supports negative patterns (e.g., `!foo.js`).

Therefore, for example, `fs.readFileSync(".gitignore", "utf8").split("\n")` is a valid value of the `ignorePatterns` property.

Reply:
Good catch. @khawarlatifkhan Do you need any additional information to fix this? There's a lot of implicit knowledge here :)

Reply:
I think I'll need some help with this so that I don't get anything wrong while editing.

Reply:
Here's a suggestion:

```suggestion
You can tell ESLint to ignore specific files and directories using `ignorePatterns` in your config files. `ignorePatterns` patterns follow the same rules as `.eslintignore`. Please see the [the `.eslintignore` file documentation](./ignoring-code.md#the-eslintignore-file) to learn more.
```